cmd/zc_attr_filter_windows.go

```diff
@@ -0,0 +1,165 @@
+// Copyright © 2017 Microsoft <wastore@microsoft.com>
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package cmd
+
+import (
+	"path/filepath"
+	"syscall"
+	"strings"
+)
+
+type excludeAttrFilter struct {
+	fileAttributes uint32
+	filePath       string
+}
+
+func (f *excludeAttrFilter) doesSupportThisOS() (msg string, supported bool) {
+	msg = ""
+	supported = true
+	return
+}
+
+func (f *excludeAttrFilter) doesPass(storedObject storedObject) bool {
+	fileName := filepath.Join(f.filePath, storedObject.name)
+	lpFileName, err := syscall.UTF16PtrFromString(fileName)
+
+	// If it fails to retrive the pointer from file path, let the filter pass
```

Comment:
@zezha-msft Any thoughts on letting it pass if there's an error?  It's kind of necessary with the current method signatures, unless we panic instead.  Panicing here doesn't seem too crazy to me, because the conversion should "never" fail.